lib/actions/interaction.js

```diff
@@ -34,7 +36,9 @@ your configuration is not in effect');
   /* eslint-enable */
 
   instance(provider).configuration('interactions').url = async function interactionUrl(ctx) {
-    return url.parse(ctx.oidc.urlFor('interaction', { uid: ctx.oidc.uid })).pathname;
+    const configuredUrl = url.parse(ctx.oidc.urlFor('interaction', { uid: ctx.oidc.uid }));
```

Comment:
can you explain why this is helpful?

Reply:
True. 
I haven't realized it's not called at all with ```devInteractions: { enabled: false }```
Thanks.